gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Fri May 22 11:56:49 SAST 2020
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.4.1-all.zip
```

Comment:
upgrade necessary or 5.5 will do for J8 ~ 12 ?

Reply:
Gradle 5.5 was giving me issues when compiling with java 8 but an update to the latest gradle version solves this, this should not have issue on the partners since 

1. This is project-specific so if they have access to GitHub this will run without affecting the environment since the wrapper is only used with gradlew not gradle command
2. This is only used for build automation and is not shipped to mavern